### PR TITLE
Navigator refactor update to latest RN version

### PR DESCRIPTION
**Pull request by markrickert**
This PR updates the `navigator-refactor` branch code and example project to work with React Native 0.26.2.

I also updated the example project. Works like a charm now. Hopefully you'll transition to this version instead of master soon... I know lots of people (including me!) really want multi-image support.

Thanks for the great library!

### CODE REVIEW

**Comment by GantMan** [comment]
woah

File: Navigator.js [hunk 1]
```diff
@@ -142,13 +142,13 @@ var LightboxNavigator = React.createClass({
...
-    var { children, ref, renderScene, configureScene, sceneStyle, ...props } = this.props;
+    var { children, reference, renderScene, configureScene, sceneStyle, ...props } = this.props;
     return (
       <Navigator
         ref={component => {
           this._root = component
-          if(typeof ref === 'function') {
-            ref(component);
+          if(typeof reference === 'function') {
+            reference(component);
```
Comment (oblador) [inline Navigator.js]: What's the reason for this change?

File: Overlay.js [hunk 2]
```diff
@@ -258,15 +260,15 @@ var LightboxOverlay = React.createClass({
...
 
-    var preview = originElement && cloneElement(
-      Children.only(originElement),
+    var preview = originElement && React.Children.map(React.Children.only(originElement), (child) => React.cloneElement(child,
       {
         style: [
           openStyle,
           { opacity: !this._isContentRendered || isAnimating ? 1 : 0 },
         ],
+        // reference: this.props.ref
```
Comment (oblador) [inline Overlay.js]: Should this be here? If not, please remove :-)
